app-ios/Modules/Package.swift

```diff
@@ -106,6 +106,7 @@ var package = Package(
             name: "Sponsor",
             dependencies: [
                 "Assets",
+                "Component",
```

Comment:
I’m curious why did the build sometimes succeed without this line? 🤔 

I wonder if the builds of each dependency were running in parallel and `Component` was sometimes built before `Sponsor` was built by accident.

Reply:
> Component was sometimes built before Sponsor was built by accident.

I think it's possible.
In last year's example, it was successful to build each module before building the app.